Feature pagination

Aayam1o1 opened the pull request.

Aayam1o1 committed:
feat! Added CRUD for parking details

Used viewset for adding parking details

Aayam1o1 committed:
Completed authentication with JWT token

Used simplejwt for tokens

Implemented abstract user for registration

Aayam1o1 committed:
Completed user based permission

Implemeneted permission based on owner employee

Owner can see only their parking details

Employee can add the car in parking space

Admin can create and add vehicles into the space

Aayam1o1 committed:
feat!: Added pagination

Created pagination using 3 ways standard, offset and cursor

NiteshTuladhar commented on parking/parking/settings.py:
```diff
@@ -82,14 +83,26 @@
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        # 'ENGINE': 'django.db.backends.sqlite3',
```
use .env to store sensitive data

Aayam1o1 committed:
Added .env file and made chaneges in ParkingSerializer

Added files like debug,jwt, secret ket and database details in .env

Made list into tuples
